hedera-node/hedera-mono-service/src/testFixtures/java/com/hedera/test/factories/scenarios/TokenCreateScenarios.java

```diff
@@ -173,6 +183,70 @@ public PlatformTxnAccessor platformTxn() throws Throwable {
+            return PlatformTxnAccessor.from(
+                    newSignedTokenCreate()
+                            .missingAdmin()
+                            .plusCustomFee(
+                                    royaltyFee(
+                                            1,
+                                            2,
+                                            new FixedFeeSpec(1, EntityId.fromNum(5L)),
+                                            collector,
+                                            false))
+                            .get());
+        }
+    },
+    TOKEN_CREATE_WITH_FIXED_FEE_DENOMINATION_AND_NO_SIG_REQ {
+        public PlatformTxnAccessor platformTxn() throws Throwable {
+            final var collector = EntityId.fromGrpcAccountId(NO_RECEIVER_SIG);
+            return PlatformTxnAccessor.from(
+                    newSignedTokenCreate()
+                            .missingAdmin()
+                            .plusCustomFee(fixedFee(123L, EntityId.fromNum(5L), collector, false))
+                            .get());
+        }
+    },
+    TOKEN_CREATE_WITH_FIXED_FEE_NO_DENOM_AND_SIG_REQ {
+        public PlatformTxnAccessor platformTxn() throws Throwable {
+            final var collector = EntityId.fromGrpcAccountId(RECEIVER_SIG);
+            return PlatformTxnAccessor.from(
+                    newSignedTokenCreate()
+                            .missingAdmin()
+                            .plusCustomFee(
+                                    royaltyFee(1, 2, new FixedFeeSpec(1, null), collector, false))
+                            .get());
+        }
+    },
+    TOKEN_CREATE_WITH_FIXED_FEE_NO_DENOM_AND_NO_SIG_REQ {
+        public PlatformTxnAccessor platformTxn() throws Throwable {
+            final var collector = EntityId.fromGrpcAccountId(NO_RECEIVER_SIG);
+            return PlatformTxnAccessor.from(
+                    newSignedTokenCreate()
+                            .missingAdmin()
+                            .plusCustomFee(
+                                    royaltyFee(1, 2, new FixedFeeSpec(1, null), collector, false))
+                            .get());
+        }
+    },
+    TOKEN_CREATE_WITH_ROYALTY_FEE_COLLECTOR_FALLBACK_WILDCARD_AND_SIG_REQ {
+        public PlatformTxnAccessor platformTxn() throws Throwable {
+            final var collector = EntityId.fromGrpcAccountId(RECEIVER_SIG);
+            return PlatformTxnAccessor.from(
+                    newSignedTokenCreate()
+                            .missingAdmin()
+                            .plusCustomFee(
+                                    royaltyFee(
+                                            1,
+                                            2,
+                                            new FixedFeeSpec(1, MISSING_ENTITY_ID),
+                                            collector,
+                                            false))
+                            .get());
+        }
+    },
```

Comment:
@tinker-michaelj @Neeharika-Sompalli can you verify these new scenarios are implemented and tested correctly?

Reply:
I think they are adding more scenarios. Seems fine to me